guest/rust/examples/ui/scroll/src/client.rs

```diff
@@ -0,0 +1,78 @@
+use ambient_api::prelude::*;
+
+#[element_component]
+fn App(hooks: &mut Hooks) -> Element {
+    let size_info = hooks.use_query(window_logical_size());
+    let x = size_info[0].1.x as f32;
+    let y = size_info[0].1.y as f32;
+    let f = FlowColumn::el([
```

Comment:
Use the `WindowSized` component instead of reading out the window size manually here

Reply:
Can you give an example to get the size of `WindowSized`?
I can manage to get the ID of `WindowSized` but it's `width()` or `height()` does not exist unless manually specified.